Project.toml

```
@@ -95,6 +96,7 @@ TimerOutputs = "0.5.7"
 Triangulate = "2.0"
 TriplotBase = "0.1"
 TriplotRecipes = "0.1"
+TrixiBase = "0.1"
```

```suggestion
TrixiBase = "0.1.1"
```
Ah, I just see that we should bump the compat to v0.1.1 because of the jldoctest bug in Trixi.jl.